requirements/edx/base.in

```diff
@@ -96,7 +96,7 @@ edx-user-state-client
 edx-when
 edxval
 event-tracking
-fs==2.0.18
+fs==2.0.26
```

Comment:
Would you be able to test unpinning this entirely, running `make compile-requirements`, and testing again? It would be great to get rid of this pin. It looks like this is a holdover from when the requirements files were reorganized 4 years ago.
```suggestion
fs
```